packages/core/js/MobileCore.js

```diff
@@ -136,22 +136,22 @@ module.exports = {
    * @param event Required parameter with {@link Event} instance to be dispatched. Should not be nil
    * @return true if the the event dispatching operation succeeded, otherwise the promise will return an error
    */
-  dispatchEvent(event: AEPExtensionEvent): Promise<boolean> {
+  dispatchEvent(event: Event): Promise<boolean> {
     return RCTAEPCore.dispatchEvent(event);
   },
 
   /**
-   * This method will be used when the provided {@code AEPExtensionEvent} is used as a trigger and a response event
+   * This method will be used when the provided {@code Event} is used as a trigger and a response event
    * is expected in return. The returned event needs to be sent using
    * {@link #dispatchResponseEvent(Event, Event, ExtensionErrorCallback)}.
    * <p>
    *
-   * @param event            required parameter, {@link AEPExtensionEvent} instance to be dispatched, used as a trigger
+   * @param event            required parameter, {@link Event} instance to be dispatched, used as a trigger
    * @param responseCallback required parameters, {@link Promise} to be called with the response event received
    *
    * @see AEPCore#dispatchResponseEvent(Event, Event, ExtensionErrorCallback)
    */
-  dispatchEventWithResponseCallback(event: AEPExtensionEvent): Promise<AEPExtensionEvent> {
+  dispatchEventWithResponseCallback(event: Event): Promise<Event> {
```

Comment:
can be renamed to: dispatchWithResponseCallback..

Unrelated to your changes, but I noticed this API overwrites the timeout to 1 sec in the objc implementation. Is this the desired behavior or can we fallback to the default in the native extension?

Reply:
https://github.com/adobe/aepsdk-core-ios/blob/daa7245798d99b9cc85f827a125d069aa7a54c29/AEPCore/Sources/core/MobileCore.swift#L118
`1 second` is the default timeout value of AEPCore API. We didn't overwrite it anyway.